client/src/Components/Map/Map.js

```diff
@@ -131,4 +132,14 @@ class MapComponent extends Component {
   }
 } // MapComponent
 
+MapComponent.propTypes = {
+  userPosition: PropTypes.object,
+  updateUserPosition: PropTypes.func,
+  SearchCountry: PropTypes.string,
+  updateCurrentCountry: PropTypes.func,
+  currentCountry: PropTypes.object,
+  user: PropTypes.object,
+  scratched: PropTypes.bool
+};
+
 export default MapComponent;
```

Comment:
I'm curious why this was needed in its current form?

Reply:
Prop-Types are a standard technique to validate the type of data that is being passed down from parent components. Prop-Types can single-handedly alert you via console log that a component is not receiving a prop of a specific type. It's handy when troubleshooting when the build breaks. Since it's a react-standard, the code editor will yell at you when you don't validate functions that are passed down from a component's parent.